### PR TITLE
CRM-16941 - Search Form and api.Get fixes for Contact

### PR DESCRIPTION
**Pull request by kurund**
- [CRM-16941: Search Form and api.Get fixes for Contact ](https://issues.civicrm.org/jira/browse/CRM-16941)

### CODE REVIEW

File: CRM/Contribute/BAO/Contribution.php [hunk 1]
```diff
@@ -2098,7 +2098,7 @@ public function loadRelatedObjects(&$input, &$ids, $required = FALSE, $loadAll =
         'payment_processor'
       );
       if ($paymentProcessorID) {
-        $intentionalEnotice = $CRM16923AnUnreliableMethodHasBeenUserToDeterminePaymentProcessorFromContributionPage;
+        //$intentionalEnotice = $CRM16923AnUnreliableMethodHasBeenUserToDeterminePaymentProcessorFromContributionPage;
```
Comment (eileenmcnaughton) [inline CRM/Contribute/BAO/Contribution.php]: Can we leave this enotice in - anything that hits an enotice on this line should trigger a bug report as it is dodgier than a clown in a dodgem

**Comment by monishdeb** [comment]
@eileenmcnaughton I have un-commented that line but handled test-case accordingly. Please have a look if it works for ya !!

**Comment by eileenmcnaughton** [comment]
Nah Monish - just merge it with that test failing - master isn't Blue anyway and it is a real (and pre-existing to your work) bug that the tests found that needs fixing. It's appropriate to leave it failing until fixed

**Comment by monishdeb** [comment]
Reverted

**Comment by kurund** [comment]
jenkins, test this please

**Comment by monishdeb** [comment]
Jenkin, test this please

**Comment by kurund** [comment]
unrelated failure

**Pull request merged**